.github/actions/free-disk-space/action.yml

```diff
@@ -0,0 +1,89 @@
+# Merge a "patch branch"
+name: 'Free disk space'
+description: 'Free runner disk space'
+runs:
+  using: "composite"
+  steps:
+    - name: Disk usage
+      shell: bash
+      run: df -h
+      # (as of 2023-08-23:)
+      # Filesystem      Size  Used Avail Use% Mounted on
+      # /dev/root        84G   62G   22G  74% /
+      # tmpfs           3.4G  172K  3.4G   1% /dev/shm
+      # tmpfs           1.4G  1.1M  1.4G   1% /run
+      # tmpfs           5.0M     0  5.0M   0% /run/lock
+      # /dev/sda15      105M  6.1M   99M   6% /boot/efi
+      # /dev/sdb1        14G  4.1G  9.0G  31% /mnt
+      # tmpfs           694M   12K  694M   1% /run/user/1001
+    - name: List docker images
+      shell: bash
+      run: docker image ls -a
+    - name: Remove unused Docker images
+      shell: bash
+      # (as of 2023-08-23:)
+      # REPOSITORY       TAG         IMAGE ID       CREATED        SIZE
+      # node             14          1d12470fa662   4 months ago   912MB
+      # node             14-alpine   0dac3dc27b1a   4 months ago   119MB
+      # node             16          ebdc80ddefea   6 days ago     909MB
+      # node             16-alpine   2573171e0124   12 days ago    118MB
+      # node             18          95d8a703ee8c   6 days ago     1.09GB
```

Comment:
if we removed only the 3 big node images, wouldnt we have enough room for a long time?